packages-exp/auth-exp/src/core/strategies/credential.ts

```diff
@@ -33,12 +31,17 @@ export async function signInWithCredential(
 ): Promise<UserCredential> {
   const auth = authExtern as Auth;
   const credential = credentialExtern as AuthCredential;
-  // TODO: handle mfa by wrapping with callApiWithMfaContext
-  const response = await credential._getIdTokenResponse(auth);
+  const operationType = OperationType.SIGN_IN;
+  const response = await _handleMfaErrors(
```

Comment:
Minor nit: I think we should find a better name for `_handleMfaErrors`. It makes it sound like that's _all_ it does, when in reality it's more of a `_performActionAndTranslateMfaErrors` or something

Reply:
I really struggle with the naming here... it's really just "catchTheErrorAndJamABunchOfStateOntoIt"

Reply:
done